plugins/modules/ipaservice.py

```diff
@@ -436,10 +586,16 @@ def main():
 
     # general
     names = ansible_module.params_get("name")
+    services = ansible_module.params_get("services")
 
     # service attributes
     principal = ansible_module.params_get("principal")
     certificate = ansible_module.params_get("certificate")
+    # A training newline is removed while adding the certificate with
+    # serive_add_cert. To be able to compare the results from service_show
+    # with the given certificates we have to remove the newline.
+    if certificate is not None:
+        certificate = [cert.rstrip('\n') for cert in certificate]
```

Comment:
I'm fine with `cert.rstrip('\n')`, but is it only `\n` that can appear, or could we use `cert.rstrip()` (or even `cert.strip()`) ?

Reply:
It should be even `cert.strip()`. I just verified with `service_add_cert`.

Reply:
Every leading or trailing white space is removed.

Reply:
Fixed